deploy/maps/action/configmap.action.application.kappnav.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kappnav.actions.application.kappnav
+data:
+  cmd-actions: |
+    [
+      {
+        "name": "inventory",
+        "text": "Inventory",
+        "text.nls": "action.cmd.kappnav.inventory.text",
+        "description": "Take inventory of all kappnav resources.",
+        "description.nls": "action.cmd.kappnav.inventory.desc",
+        "image": "docker.io/cvignola/kappnav-inv:latest",
```

Comment:
Guessing we don't actually want to hard-code the name of the image. Is there a default location? There's already an extension mechanism in place in the config to address additional images.

Reply:
For example:
"image": "{{ .Spec.ExtensionContainers.kappnavInv.Repository }}:{{ .Spec.ExtensionContainers.kappnavInv.Tag }}"

Reply:
Good, good catch.  Yes, we can't have a hard-coded image.  Especially that one!